packages/react-widget/src/VoucherifyRedeem.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react'
```

Comment:
In old SDK `amount` field is not actually required to be entered even after enabling it.